src/modules/account/components/MaterialRequestDetailBlade/MaterialRequestDetailBlade.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button } from '@meemoo/react-components';
+import clsx from 'clsx';
+import Image from 'next/image';
+import React, { FC } from 'react';
+
+import { Blade, Icon, IconNamesLight } from '@shared/components';
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+import { formatMediumDate } from '@shared/utils';
+
+import styles from './MaterialRequestDetailBlade.module.scss';
+
+import { MATERIAL_REQUEST_TRANSLATIONS_BY_TYPE } from '@material-requests/const';
+import { MaterialRequestDetail } from '@material-requests/types';
+
+interface MaterialRequestDetailBladeProps {
+	isOpen: boolean;
+	onClose: () => void;
+	currentMaterialRequestDetail?: MaterialRequestDetail;
+}
+
+const MaterialRequestDetailBlade: FC<MaterialRequestDetailBladeProps> = ({
+	isOpen,
+	onClose,
+	currentMaterialRequestDetail,
+}) => {
+	const { tText } = useTranslation();
+
+	const renderFooter = () => {
+		return (
+			<div className={styles['p-account-my-material-requests__close-button-container']}>
+				<Button
+					label={tText(
+						'modules/account/components/material-request-detail-blade/material-requests___sluit'
+					)}
+					variants={['block', 'text']}
+					onClick={onClose}
+					className={styles['p-account-my-material-requests__close-button']}
+				/>
+			</div>
+		);
+	};
+
+	return (
+		<Blade
+			isOpen={isOpen}
+			renderTitle={() => (
+				<h2 className={styles['p-account-my-material-requests__title']}>
+					{tText(
+						'modules/account/components/material-request-detail-blade/material-requests___detail'
+					)}
+				</h2>
+			)}
+			footer={isOpen && renderFooter()}
+			onClose={onClose}
+		>
+			<div className={styles['p-account-my-material-requests__maintainer']}>
+				{currentMaterialRequestDetail?.maintainerLogo && (
+					<div className={styles['p-account-my-material-requests__maintainer-logo']}>
+						<Image
+							alt="maintainer logo"
+							src={currentMaterialRequestDetail?.maintainerLogo || ''}
```

Comment:
currentMaterialRequestDetail?.maintainerLogo is already checked before rendering this section, so you can just put:
```suggestion
							src={currentMaterialRequestDetail.maintainerLogo}
```